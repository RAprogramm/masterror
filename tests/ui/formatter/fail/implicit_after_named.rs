// SPDX-FileCopyrightText: 2025 RAprogramm <andrey.rozanov.vl@gmail.com>
//
// SPDX-License-Identifier: MIT

use masterror::Error;

#[derive(Debug, Error)]
#[error("{0}: {}", self.first, self.second)]
struct ImplicitAfterNamedError {
    first:  &'static str,
    second: &'static str,
}

fn main() {}
