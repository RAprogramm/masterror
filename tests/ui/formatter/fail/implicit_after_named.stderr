error: multiple unused formatting arguments
 --> tests/ui/formatter/fail/implicit_after_named.rs:7:17
  |
7 | #[derive(Debug, Error)]
  |                 ^^^^^
  |                 |
  |                 multiple missing formatting specifiers
  |                 argument never used
  |                 argument never used
  |
  = note: this error originates in the derive macro `Error` (in Nightly builds, run with -Z macro-backtrace for more info)
